configs/PyTorchLightning/flash_pl-master.yaml

```diff
@@ -13,6 +13,9 @@ target_repository:
     - tests/examples/utils.py
     - tests/examples/test_scripts.py
 
+before_install:
+  - sudo apt-get install -y libsndfile1
```

Comment:
with this, you limit it only to Linux machines...
if you feel it is worse to have this before for all OS, let's create an issue and I'll check it later :rabbit:

Reply:
Have tried to limit to just image,text,tabular for now which should fix it. We can revisit the audio stuff later :smiley: